btree/btree.c

```diff
@@ -2063,6 +2101,20 @@ static int ff_valsize(const struct nd *node)
 	return ff_data(node)->ff_vsize;
 }
 
+static bool ff_underflow(const struct nd *node, bool predict)
+{
+	uint16_t rec_count = ff_data(node)->ff_used;
+	if (predict)
+		rec_count--;
+	return  rec_count == 0 ? true : false;
+}
+
+static bool ff_overflow(const struct nd *node)
+{
```

Comment:
Isn't it opposite of `ff_isfit()` ? Can't we use that ?

Reply:
For fixed format node, it may work, but I guess It is better to keep overflow function separately as we might want to modify definition for overflow,

Reply:
Ok.